port/action/schema.go

```diff
@@ -678,7 +677,7 @@ func ObjectPropertySchema() schema.Attribute {
 			},
 		},
 		"encryption": schema.StringAttribute{
-			MarkdownDescription: "The algorithm to encrypt the property with",
+			MarkdownDescription: "The algorithm to encrypt the property with. Accepted value: `aes256-gcm`, `aes128-gcm`",
 			Optional:            true,
 			Validators: []validator.String{
 				stringvalidator.OneOf("aes256-gcm"),
```

Comment:
seems like only one is expected not two, please correct me if im wrong

Reply:
True. the other value is also accepted but let me take out the other one to prevent misleading. since just one is accepted by the validator